surelock/src/main/java/com/smashingboxes/surelock/SurelockDefaultDialog.java

```diff
@@ -61,24 +74,70 @@ public void onCreate(Bundle savedInstanceState) {
 
         // Do not create a new Fragment when the Activity is re-created such as orientation changes.
         setRetainInstance(true);
-        setStyle(DialogFragment.STYLE_NO_TITLE, android.R.style.Theme);
+
+        TypedArray attrs = getActivity().obtainStyledAttributes(styleId, R.styleable
+                .SurelockDefaultDialog);
+        int dialogTheme = attrs.getResourceId(R.styleable.SurelockDefaultDialog_sl_dialog_theme,
+                Integer.MAX_VALUE);
```

Comment:
I had been using `-1`, but the color white's int value is `-1`. 🤦‍♂️ I'll change it to `0`